Vagrantfile

```diff
@@ -11,16 +11,23 @@ Vagrant.configure(2) do |config|
     config.vbguest.auto_update = true
   end
 
-  config.vm.define "dcos-centos-virtualbox" do |vm_cfg|
-    vm_cfg.vm.box = $dcos_box
-    vm_cfg.vm.box_url = $dcos_box_url
-    vm_cfg.vm.box_version = $dcos_box_version
+  config.vm.hostname = 'dcos-centos'
+  config.vm.box = $dcos_box
+  config.vm.box_url = $dcos_box_url
+  config.vm.box_version = $dcos_box_version
 
-    vm_cfg.vm.provider "virtualbox" do |v|
-      v.name = "dcos-centos-virtualbox"
-      v.cpus = 2
-      v.memory = 2048
-      v.customize ['modifyvm', :id, '--natdnshostresolver1', 'on']
-    end
+  config.vm.provider "virtualbox" do |vb|
+    vb.name = "dcos-centos-virtualbox"
+    vb.cpus = 2
+    vb.memory = 2048
+    vb.customize ['modifyvm', :id, '--natdnshostresolver1', 'on']
+    vb.linked_clone = true
   end
+  config.vm.provider "vmware_fusion" do |vmware|
+    vmware.vmx['memsize'] = 2048
```

Comment:
I guess that the memory used in `vmware` is purposefully the same as in `vb` (and same with `cpus`. If so, can we use a variable?